assets/js/contact.js

```diff
@@ -9,8 +17,16 @@ function verifyCanSubmitEntry() {
   const piiError = document.getElementById('pii-warning');
   const piiErrorText = document.getElementById('pii-warning-message');
   const descriptionInput = document.getElementById('description');
+  const emailInput = document.getElementById('email');
+  const spamEmailDigests = (form.dataset.spamEmailAddresses || '').split(',').filter(Boolean);
+
   let alreadyAttemptedSubmission = false;
   form.addEventListener('submit', (event) => {
+    if (spamEmailDigests.includes(sha256Hex(emailInput.value))) {
```

Comment:
Based on the package documentation, looks like you could also call `sha256` directly.

```suggestion
    if (spamEmailDigests.includes(sha256(emailInput.value))) {
```

Reply:
great catch, thanks!